### ISSUE TITLE
Keyboard navigation fails selecting command line or OS

### ISSUE THREAD

**Issue by 24sandbrookt-coder** [nautilus-os/NautilusOS#123]
None of those work when trying to select cmd line or the os

**Comment by 24sandbrookt-coder** [comment]
thru the html unzipped in the folder

**Comment by CompactAIOfficial** [comment]
Re-download the .html file (I assume the NautilusOS-onefile), click into the window, and use the arrow keys. Works fine for me.
